source/delegate/contracts/Delegate.sol

```diff
@@ -415,13 +415,15 @@ contract Delegate is IDelegate, Ownable {
     address signerToken
   ) internal {
 
-    // Delete the rule.
-    delete rules[senderToken][signerToken];
-
-    emit UnsetRule(
-      owner(),
-      senderToken,
-      signerToken
+    // using non-zero rule.priceCoef for rule existence check
```

Comment:
Do you think we should add a require on _setRule for
`require(priceCoef > 0)` ?

Reply:
Yes, if someone wants to do a priceCoef == 0, they honestly should not use the Delegate to do this.